assets/addon_browser/addons.conf

```diff
@@ -487,5 +487,14 @@
       "modrinth": "https://modrinth.com/mod/bluemap-sign-markers"
       "github": "https://github.com/tpwalke2/BlueMapSignMarkers"
     }
+  {
+    name: "BlueMap Skin Integration"
+    description: "A BlueMapAPI V2 plugin to add support for additional skin providers."
```

Comment:
It would be useful to mention all the skin providers that this plugin supports in the description, so people using the search function on the addons list page can find your plugin :)